fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -1577,4 +1580,31 @@ public TStorageFormat getStorageFormat() {
         }
         return tableProperty.getStorageFormat();
     }
+
+    public boolean satisfyHashDistribution(List<Expr> hashExprs) {
```

Comment:
Rename to `couldLocalGorupBy` or similar name?

Reply:
Rename to `meetAggDistributionRequirements` and change the param to `AggregateInfo`.